### ISSUE TITLE
Questions Regarding Experimental Settings

### ISSUE THREAD

**Issue by xidao4** [graphdeco-inria/gaussian-splatting#82]
Dear Authors, I'm reaching out to you with some questions about the experimental settings described in your paper, as I've been unable to replicate the accuracy results. I have two specific questions:

In the paper, you mention that "we periodically remove Gaussians that are very large in worldspace and those that have a big footprint in viewspace." In the source code, the threshold for removing large points in viewspace is set to 20. Could you please clarify what threshold value was used in the experiments that achieved the accuracy reported in the paper?

The paper also states that you "clone the Gaussians by simply creating a copy of the same size, and moving it in the direction of the positional gradient." However, the source code doesn't seem to include the movement of the new points. Was this operation performed in the experiments that led to the paper's accuracy?

I would like to express my gratitude for your work and look forward to your response to help us better understand the implementation details. Thank you!

**Comment by Snosixtyboo** [comment]
Hi,

could you please tell us
- the setup you are running on (GPU model and CUDA/Pytorch version)
- which scene you are testing (and how you are running it)
- which accuracy (PSNR, SSIM) you fail to obtain, and by how much?

the source code does indeed not **explicitly** move the Gaussians after cloning. That movement occurs because the positional gradient for half of the clones is **not** set to 0, so when we do the optimizer step, one of the two clones just stays in place, the other does a gradient descent update.

Hth,
Bernhard

**Comment by xidao4** [comment]
Hi Bernhard, 

- I'm using Tesla V100-PCIE, cuda 11.3,  torch 1.12.1 and use the environment.yml to create the virtual environment.

- I'm testing Garden with the following command 

```
python train.py -s /dir/to/garden/ --eval
python render.py --skip_train -m /output/dir/to/garden
python metrics.py -m  /output/dir/to/garden
```

- And we get the result: SSIM 0.8559532, PSNR 27.2766361, LPIPS 0.1223336
In paper, the result is SSIM 0.868, PSNR 27.410, LPIPS 0.103

Is the threshold for removing large points in viewspace related to different scenes?

Thanks a lot!

**Comment by Snosixtyboo** [comment]
Ok thanks. Couple of things here:

- We evaluated the mipnerf360 outdoor scenes with ```-i images_4``` to be consistent with the mipnerf360 paper (cf. ```full_eval.py```)
- We did a major overhaul on the code for the release. Some scenes can have slightly lower quality because of this, but on average the scores should now be higher than what we report in the paper. If you run the metrics script on our pre-computed images, you should get the exact numbers from the paper.
- If you are doing a scientific evaluation, feel free to compare against the numbers you prefer: the ones from the original paper or the ones from the code release (which **should** be slightly better on average). 

Hth,
Bernhard
